src/api/v4/filter/bounding-box-leaflet.js

```diff
@@ -53,4 +56,12 @@ BoundingBoxLeaflet.prototype.$getInternalModel = function () {
   return this._internalModel;
 };
 
+// Helper to check if an element is a leafletmap object
+function _isLeafletMap (element) {
```

Comment:
I would use `instance of L.Map` to skip also Bool, Number, String, Array and wrong Objects.